packages/core/android/src/main/kotlin/com/datadog/reactnative/DatadogSDKWrapper.kt

```diff
@@ -21,6 +26,45 @@ import com.datadog.android.trace.Trace
 import com.datadog.android.trace.TraceConfiguration
 import com.datadog.android.webview.WebViewTracking
 
+/**
+ * Internal object used to add internal testing.
+ */
+object DatadogSDKWrapperStorage {
```

Comment:
If we don't want to make it available to the customers, we need to make this class `internal`.

```suggestion
internal object DatadogSDKWrapperStorage {
```

Reply:
If I make it `internal`, I cannot import it in the internal testing module.
Is it possible to still import it?

And I think it can be ok to leave it available to customers. Most of them won't do anything, and maybe that can make things a little easier for some customers with hybrid applications.

Reply:
Let's keep it public then, but if it used only for the tests, nothing in the name reflects that the intended usage is for the tests only.

Reply:
It will also be used in Session Replay to make sure we use the same core for all features - it's one of the next tasks on the topic. 
When we'll split the RN SDK in core/RUM/Logs/Trace this will also be used to get the core in the feature packages.

Let me know if you think we should still change the name to something more explicit here.